integrations/sveltekit/src/api-reference.ts

```diff
@@ -0,0 +1,36 @@
+import { getHtmlDocument } from '@scalar/core/libs/html-rendering'
+
+import { customTheme } from './custom-theme.js'
+import type { ApiReferenceConfiguration } from './types.js'
```

Comment:
same as above about regarding the file extension

```suggestion
import type { ApiReferenceConfiguration } from './types.ts'
```